FreeRTOS/FreeRTOS-Kernel
# POSIX Port: Remove pthread_attr_setstacksize call

Description
------------
We have removed the use of  `pthread_attr_setstack` and as a result, the task stack is no longer used as the corresponding pthread's stack. There is no use of calling `pthread_attr_setstacksize` as the default is always good enough and we don't need to handle OS specific cases. 

This PR simplifies the code by removing the call to `pthread_attr_setstacksize`.

Test Steps
-----------
I had a task with a 4kb stack which was causing the issue. Simply `xTaskCreate` with a 4kb stack on macOS and it will crash.

Checklist:
----------

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.
